docs/tutorials/vertical-fl.md

```diff
@@ -43,6 +43,11 @@ The data format for VFL is different from regular FL. That is why each of our ex
 
 Please follow steps in [CCFRAUD - Add your Kaggle credentials to the workspace key vault](../real-world-examples/ccfraud.md#Add-your-Kaggle-credentials-to-the-workspace-key-vault). Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example CCFRAUD_VERTICAL`**).
 
+
+### BANK_MARKETING
+
+This example is similar to **CCFRAUD**, however here also host owns part of the feature space. Please follow steps in [BANK_MARKETING - Add your Kaggle credentials to the workspace key vault](../real-world-examples/bank-marketing.md#Add-your-Kaggle-credentials-to-the-workspace-key-vault). Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example BANK_MARKETING`**).
```

Comment:
I did not find `--example MNIST_VERTICAL`. Also, is it `BANK_MARKETING_VERTICAL`?